source/manual/databases.html.md

```diff
@@ -0,0 +1,37 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Databases on GOV.UK
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+---
+
+Many GOV.UK applications use a database. See our [spreadsheet of applications and database engines](https://docs.google.com/spreadsheets/d/1rBZeeaT9XevaRvstMjw2YsbSsoB7IqgWjpJz5oAO9nM/edit#gid=1368371571) for details.
+
+## Database engines
+
+GOV.UK uses mostly PostgreSQL and some MySQL for apps that require a relational database. There's no particular reason why we use both, and the fact that we do is considered [tech debt](https://trello.com/c/zlfgSJlV/69-govuk-uses-mysql-and-postgresql-and-mongo).
+
+GOV.UK uses MongoDB for apps that require a document database. MongoDB is synonymous with DocumentDB; the name describes where how it is hosted, but the underlying engine is the same.
```

Comment:
[Mongo disagree](https://www.mongodb.com/atlas-vs-amazon-documentdb#:~:text=Amazon%20DocumentDB%20is%20a%20NoSQL,of%20Amazon's%20Aurora%20backend%20platform.) with the "MongoDB is synonymous with DocumentDB" bit. I sort of agree with the sentiment, but it might need softening a bit.